console/root.go

```diff
@@ -99,6 +99,7 @@ func (c *Component) Start() error {
 	endpoint.GET("/filter/saved", c.filterSavedListHandlerFunc)
 	endpoint.DELETE("/filter/saved/:id", c.filterSavedDeleteHandlerFunc)
 	endpoint.POST("/filter/saved", c.filterSavedAddHandlerFunc)
+	endpoint.POST("/context/table-interval", c.getTableAndIntervalHandlerFunc)
```

Comment:
`/graph/table-interval` is better I think. And, please, add a test for this endpoint.